plugins/nf-sqldb/src/main/nextflow/sql/QueryHandler.groovy

```diff
@@ -156,10 +159,29 @@ class QueryHandler implements QueryOp<QueryHandler> {
     protected void query0(Connection conn) {
         try {
             try (Statement stm = conn.createStatement()) {
-                try( def rs = stm.executeQuery(normalize(statement)) ) {
-                    if( emitColumns )
-                        emitColumns(rs)
-                    emitRowsAndClose(rs)
+                final String normalizedStmt = normalize(statement)
```

Comment:
well, `fromQuery` was meant to carry out .. queries! 😄 would not make more sense to add one or more functions (not operators) to executor DDL or other statements?

Reply:
Agreed, I just wanted to make it work as is!

I'll make another PR with that approach to see how it'll look.